<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Destino Y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Velocidad:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>230</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rojo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Verde:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Azul:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="destX_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="destY_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="speed_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="red_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="green_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="blue_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>290</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="particle_PlainText">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>321</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addToStart_pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar Inicio</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addEnd_pushButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>340</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar Final</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showListParticle_pushButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>380</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>MOSTRAR</string>
       </property>
      </widget>
      <widget class="QLabel" name="originX_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="originY_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen Y:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="originY_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="originX_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="originX_label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="id_spinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
